docs/rules/named-functions-in-promises.md

```diff
@@ -2,6 +2,19 @@
 
 ### Rule name: `named-functions-in-promises`
 
+#### Configuration
+
+```
+ember/named-functions-in-promises: [2, {
+  allowConciseArrow: false,
```

Comment:
I'm thinking about renaming this setting to `allowSimpleArrowFunction` - I think it would be more descriptive. What do you think? :)